fastlane/actions/gh_pages.rb

```diff
@@ -14,26 +14,33 @@ def self.run(params)
         UI.user_error!("Missing `_versions` file") unless File.exists?("_versions")
         UI.user_error!("Missing `.jazzy.yaml` file") unless File.exists?(".jazzy.yaml")
         module_version = YAML.load(File.read(".jazzy.yaml"))["module_version"]
+        array = module_version.split(/[.]/)
+        short_version = array[0] + "." + array[1]
+        module_version = short_version
         UI.user_error!("Missing `module_version` parameter in .jazzy.yaml file") unless module_version != nil
+        module_name = YAML.load(File.read(".jazzy.yaml"))["module"]
+        UI.user_error!("Missing `module` parameter in .jazzy.yaml file") unless module_name != nil
+        docs_folder = "artifacts/docs"
 
         # Prepare environment
-        FileUtils.rm_rf("artifacts/docs")
-        sh "git clone --single-branch --branch gh-pages #{ghpages_url} artifacts/docs"
-        FileUtils.rm_rf("artifacts/docs/#{module_version}")
+        FileUtils.rm_rf(docs_folder)
+        sh "git clone --single-branch --branch gh-pages #{ghpages_url} #{docs_folder}"
+        FileUtils.rm_rf("#{docs_folder}/#{module_version}")
 
         # Download theme and generate docs
+        UI.user_error!("svn not installed") unless system("command -v svn")
         sh "svn export https://github.com/rakutentech/ios-buildconfig/trunk/jazzy_themes jazzy_themes --force"
-        sh "bundle exec jazzy --output artifacts/docs/#{module_version} --theme jazzy_themes/apple_versions"
+        other_action.documentation(module_name: module_name, module_version: module_version)
```

Comment:
Does this action set `--theme jazzy_themes/apple_versions` custom theme ?

Reply:
Nope, reverted this accidental change